frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -42,6 +42,7 @@ action.GitMachete.BaseFastForwardMergeBranchToParentAction.description=Fast-forw
 action.GitMachete.BaseFastForwardMergeBranchToParentAction.notification.title.ff-fail=Fast-forward merge branch ''{1}'' to ''{0}'' failed
 action.GitMachete.BaseFastForwardMergeBranchToParentAction.notification.title.ff-success=Branch <b>{1}</b> has been fast-forward merged to <b>{0}</b>
 action.GitMachete.BaseFastForwardMergeBranchToParentAction.task-title=Fast-forward merging
+action.GitMachete.BaseFastForwardMergeBranchToParentAction.task-subtitle=Fast-forwarding...
```

Comment:
or rather `fast-forward merging...` to be consistent with the recent clarification?

Reply:
Actually, I'd leave as is, coz the main action title is `Fast-forward merging` and the same subtitle bellow it IMHO can look strange, WDYT?

Reply:
Ok, let's leave as is